tests/Unit/DataStructures/DataBox/Test_TagTraits.cpp

```diff
@@ -21,6 +21,23 @@ static_assert(not db::is_compute_tag_v<
                   TestHelpers::db::Tags::Label<TestHelpers::db::Tags::Simple>>,
               "Failed testing is_compute_tag");
 
+static_assert(not db::is_simple_tag_v<TestHelpers::db::Tags::Bad>,
+              "Failed testing is_simple_tag");
```

Comment:
I think these assert messages are redundant and you can just remove them. (not requesting this change)

Reply:
I agree and have not been adding the string for new `static_assert`s where they simply repeat the conditional, and sometimes removing them.